beacon/valcheck/attestation.go

```diff
@@ -17,9 +17,8 @@ func (v *AttestationValueCheck) Check(value []byte) error {
 		return errors.Wrap(err, "could not parse input value storing attestation data")
 	}
 
-
-	if inputValue.Slot == 0 {
-		return errors.New("this is an example test error")
+	if inputValue.Slot == 100 {
```

Comment:
Is this part of the productive code or just some check to validate functionality?

Reply:
Also why 100?

Reply:
for now it's really just a placeholder that we will need to refactor.